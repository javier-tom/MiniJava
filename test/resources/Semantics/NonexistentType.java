class Main {
    public static void main(String[] args) {
        System.out.println(new A().run());
    }
}

class A {
    public int run() {
        return 17;
    }

    public Foo stuff() {
        return 5;
    }

    public int bad(Bad b) {
        return 17;
    }
}
