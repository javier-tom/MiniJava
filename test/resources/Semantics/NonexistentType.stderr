Error on line 12: unknown type Foo
Error on line 16: unknown type Bad